src/components/authorize/claims-config.ts

```diff
@@ -0,0 +1,38 @@
+export const EXPECTED_CLIENT_ID = "orchestrationAuth";
+
+export function getKnownClaims(): {
+  [key: string]: string | boolean | number;
+} {
+  return {
+    client_id: "UNKNOWN",
+    aud: "UNKNOWN",
+  };
+}
+
+// Single source of truth for claims type and claims object keys
+
+export type Claims = typeof claimsInstance;
+// construct object for type
+const claimsInstance = getClaimsObject();
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function getClaimsObject() {
+  return {
+    iss: "",
+    aud: "",
+    exp: NaN,
```

Comment:
Genuine question - what's the deal with NaN as opposed to null or undefined to indicate uninitialized values (Slack if easier and you have time to explain)

Reply:
NaN give a number type which is needed when converting this object to a type.

Reply:
Cool - I guess that makes sense. Typescript :(